### PR TITLE
simplify terminal executor

### PR DESCRIPTION
**Pull request by grosser**
@zendesk/samson

### CODE REVIEW

**Comment by irwaters** [comment]
I should add an on 💚  to that.

File: app/models/terminal_executor.rb [hunk 1]
```diff
@@ -59,35 +59,20 @@ def resolve_secrets(command)
...
+    options = {in: '/dev/null', unsetenv_others: true}
+    output, input, @pid = PTY.spawn(whitelisted_env, command, options)
+
+    @pgid = begin
+      Process.getpgid(@pid)
+    rescue Errno::ESRCH
+      nil
     end
+
+    output.each(256) { |line| @output.write line }
```
Comment (jonmoter) [inline app/models/terminal_executor.rb]: We don't need the `gsub` for `\r\n` anymore?

**Comment by grosser** [comment]
something shady going on ... worked fine on mac :/

**Comment by grosser** [comment]
nope! :D

**Pull request merged**